src/redux/books/actionTypes.js

```diff
@@ -0,0 +1,2 @@
+export const BOOK_ADDED = 'bookAdded';
+export const BOOK_REMOVED = 'bookRemoved';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: 
```
const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```